tests/e2e/scale/noobaa/test_scale_namespace_crd.py

```diff
@@ -43,8 +44,54 @@ class TestScaleNamespace(E2ETest):
                         "type": "Single",
                         "namespacestore_dict": {"aws": [(1, None)]},
                     },
-                }
+                },
+                marks=[pytest.mark.polarion_id("OCS-2518")],
             ),
+            pytest.param(
+                {
+                    "interface": "OC",
+                    "namespace_policy_dict": {
+                        "type": "Single",
+                        "namespacestore_dict": {"azure": [(1, None)]},
+                    },
+                },
+                marks=[pytest.mark.polarion_id("OCS-2558")],
+            ),
+            pytest.param(
+                {
+                    "interface": "OC",
+                    "namespace_policy_dict": {
+                        "type": "Single",
+                        "namespacestore_dict": {"rgw": [(1, None)]},
+                    },
+                },
+                marks=[
+                    on_prem_platform_required,
+                    pytest.mark.polarion_id("OCS-2559"),
+                ],
+            ),
+            pytest.param(
+                {
+                    "interface": "OC",
+                    "namespace_policy_dict": {
+                        "type": "Cache",
+                        "ttl": 60000,
+                        "namespacestore_dict": {"aws": [(1, "eu-central-1")]},
```

Comment:
Should this (eu-central-1) not be hard-coded?  Seems like it should either be configurable or set from a Constant.
Minor nit on my part, maybe.

Reply:
At the moment, all MCG-related AWS regions are hardcoded throughout OCS-CI due to two reasons:
1. Tests had to be implemented under particular time constraints, which prevented us from implementing a config system originally
2. We discussed the matter internally, and reached the conclusion that, for the time being, it'll be safe enough to rely on AWS to not change any region names; And if such change was to occur, it'd be very easy to refactor and fix in the future.

The issue is still open, and was discussed in #3338. I hope to fix it in the upcoming weeks.

Reply:
@MeridianExplorer Thanks Ben!  At this time, we will use AWS regions a hardcoded until the issue is fixed.